examples/docking-protein-protein/docking-protein-protein.cfg

```diff
@@ -55,5 +55,8 @@ noecv = true
 [caprieval]
 reference_fname = 'data/e2a-hpr_1GGR.pdb'
 
+[clustfcc]
+threshold = 1
```

Comment:
Why threshold = 1? To make sure you are getting clusters? Why not implement the automatic reduction of the threshold until you do find clusters (as done in 2.4)?

Reply:
Good point! We'll do that in another issue: https://github.com/haddocking/haddock3/issues/253